system/src/system_publish_vitals.cpp

```diff
@@ -7,11 +7,14 @@
 #include "system_cloud.h"
 #include "system_threading.h"
 
+#ifdef UNIT_TEST
+#include "spark_wiring_timer.h"
```

Comment:
This header should be out of UNIT_TEST conditional.

Reply:
my bad.

Reply:
thanks.